Makefile

```diff
@@ -71,6 +71,14 @@ endif
 
 LIBXXH = libxxhash.$(SHARED_EXT_VER)
 
+XXHSUM_SRC_DIR = cli
```

Comment:
Yes, much better.
These variables really help maintenance and readability of the `Makefile` script